coinmetrics-bytecoin/CoinMetrics/Bytecoin.hs

```diff
@@ -0,0 +1,193 @@
+{-# LANGUAGE DeriveGeneric, OverloadedLists, OverloadedStrings, StandaloneDeriving, TemplateHaskell, TypeFamilies, ViewPatterns #-}
+
+module CoinMetrics.Bytecoin
+  ( Bytecoin(..)
+  , BytecoinBlock(..)
+  , BytecoinTransaction(..)
+  , BytecoinTransactionInput(..)
+  , BytecoinTransactionOutput(..)
+  ) where
+
+import qualified Data.Aeson as J
+import qualified Data.Aeson.Types as J
+import qualified Data.ByteString as B
+import qualified Data.ByteString.Lazy as BL
+import qualified Data.ByteString.Short as BS
+import qualified Data.Text.Encoding as T
+import qualified Data.HashMap.Lazy as HML
+import qualified Data.Vector as V
+import Data.Time.Clock.POSIX
+import Data.Maybe
+import Data.Int
+import Data.Proxy
+import Hanalytics.Schema
+
+import CoinMetrics.BlockChain
+import CoinMetrics.JsonRpc
+import CoinMetrics.Schema.Flatten
+import CoinMetrics.Schema.Util
+import CoinMetrics.Util
+
+newtype Bytecoin = Bytecoin JsonRpc
+
+data BytecoinBlock = BytecoinBlock
+  { bc_difficulty :: {-# UNPACK #-} !Int64
```

Comment:
I suggest to use prefix `bb_` meaning "Bytecoin block", to be consistent with other exporters.